openwisp_monitoring/monitoring/migrations/0010_rename_threshold_alert_settings.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.0.3 on 2020-05-30 19:12
```

Comment:
Please squash all the migrations unless there is a good reason not to

Reply:
This is the reason why I can't simply squash the migrations for this PR https://github.com/openwisp/openwisp-monitoring/pull/112#discussion_r433397418

Reply:
fair enough! :+1: